tests/ha/sle15_workarounds.pm

```diff
@@ -0,0 +1,49 @@
+# SUSE's openQA tests
+#
+# Copyright (c) 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Add some SLE15 workarounds
+#          Should be removed after SLE15 will be released!
+# Maintainer: Loic Devulder <ldevulder@suse.com>
+
+use base "opensusebasetest";
+use strict;
+use utils;
+use testapi;
+use hacluster;
+
+# Install missing patterns/packages for HA support in SLE15
+sub run {
+    return unless sle_version_at_least('15');
+
+    # Modify the device number if needed
+    if (get_var('ISO_1', '') ne '') {
+        assert_script_run "sed -i 's;sr1;sr0;g' /etc/zypp/repos.d/*";
+    }
+
+    # Install ha_sles pattern if missing
+    if (script_run 'zypper pt -i | grep -q ha_sles') {
+        record_soft_failure 'bsc#1060472';
+        zypper_call("in -l -t pattern ha_sles", exitcode => [0, 102], timeout => 900);
+    }
+
+    # Install ocfs2-kmp-default package if missing
+    if (script_run 'rpm -q ocfs2-kmp-default >/dev/null') {
+        record_soft_failure 'bsc#1060601';
+        zypper_call("in -l ocfs2-kmp-default", timeout => 900);
+    }
+
+    # Install yast2-storage-ng package if missing
+    if (script_run 'rpm -q yast2-storage-ng >/dev/null') {
+        record_soft_failure 'bsc#1062009';
```

Comment:
that bug is RESOLVED FIXED since 2017-10-24, can you make sure the bug is VERIFY FIXED and them remove the workaround?

Reply:
I checked on the last run tests and yes it has been corrected, so I will remove it.

In fact, all my sle15 workarounds are now resolved, so I will just simply remove the sle15_workaround.pm script.